### PR TITLE
Add more features for iFlight_F745_AIO board.

### PR DESCRIPTION
**Pull request by Linjieqiang**
Reopen it. https://github.com/betaflight/unified-targets/pull/418

### CODE REVIEW

File: configs/default/IFRC-IFLIGHT_F745_AIO.config [hunk 1]
```diff
@@ -107,27 +118,42 @@ dma pin A03 0
 
 # feature
 feature -RX_PARALLEL_PWM
+feature -AIRMODE
```
Comment (mikeller) [inline configs/default/IFRC-IFLIGHT_F745_AIO.config:121]: Why is airmode disabled here? This is a tuning setting, and not part of the hardware configuration.

File: configs/default/IFRC-IFLIGHT_F745_AIO.config [hunk 2]
```diff
@@ -107,27 +118,42 @@ dma pin A03 0
...
 led 3 9,11::CTOBVIW:0
 
+# aux
+aux 0 0 0 900 1300 0 0
+
 # master
 set baro_bustype = I2C
 set baro_i2c_device = 1
-set baro_hardware = BMP280
+set baro_hardware = DPS310
```
Comment (mikeller) [inline configs/default/IFRC-IFLIGHT_F745_AIO.config:146]: Not sure why you are setting this here - leaving this on the default (`AUTO`) will get the device properly.

File: configs/default/IFRC-IFLIGHT_F745_AIO.config [hunk 3]
```diff
@@ -107,27 +118,42 @@ dma pin A03 0
...
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set mag_hardware = HMC5883
```
Comment (mikeller) [inline configs/default/IFRC-IFLIGHT_F745_AIO.config:149]: Not sure why you are setting this here - leaving this on the default (`AUTO`) will get the device properly.

File: configs/default/IFRC-IFLIGHT_F745_AIO.config [hunk 4]
```diff
@@ -1,3 +1,4 @@
+
```
[the remark this answers is not in the document]
Comment (Linjieqiang) [reply configs/default/IFRC-IFLIGHT_F745_AIO.config]: Fixed it. Thanks.

File: configs/default/IFRC-IFLIGHT_F745_AIO.config (hunk 1 above)
Comment (Linjieqiang) [reply configs/default/IFRC-IFLIGHT_F745_AIO.config]: Fixed it. Thanks.

File: configs/default/IFRC-IFLIGHT_F745_AIO.config (hunk 3 above)
Comment (Linjieqiang) [reply configs/default/IFRC-IFLIGHT_F745_AIO.config]: Fixed it. Thanks.

File: configs/default/IFRC-IFLIGHT_F745_AIO.config (hunk 2 above)
Comment (Linjieqiang) [reply configs/default/IFRC-IFLIGHT_F745_AIO.config]: Fixed it. Thanks.

File: configs/default/IFRC-IFLIGHT_F745_AIO.config (hunk 4 above)
[the remark this answers is not in the document]
Comment (Linjieqiang) [reply configs/default/IFRC-IFLIGHT_F745_AIO.config]: Fixed it. Thanks.

File: configs/default/IFRC-IFLIGHT_F745_AIO.config [hunk 5]
```diff
@@ -108,26 +118,40 @@ dma pin A03 0
...
+serial 6 2 115200 57600 0 115200
 
 # led
 led 0 6,11::CTOBVIW:0
 led 1 7,11::CTOBVIW:0
 led 2 8,11::CTOBVIW:0
 led 3 9,11::CTOBVIW:0
 
+# aux
+aux 0 0 0 900 1300 0 0
```
Comment (etracer65) [inline configs/default/IFRC-IFLIGHT_F745_AIO.config:139]: Seems like a very bad idea to pre-configure an arming switch. You have no way of knowing the user's radio setup and aux1 might be used for something else causing an unsafe situation.

File: configs/default/IFRC-IFLIGHT_F745_AIO.config [hunk 6]
```diff
@@ -108,26 +118,40 @@ dma pin A03 0
...
+
 # master
 set baro_bustype = I2C
 set baro_i2c_device = 1
-set baro_hardware = BMP280
+set baro_hardware = AUTO
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set mag_hardware = AUTO
+set align_mag = CW180
```
Comment (etracer65) [inline configs/default/IFRC-IFLIGHT_F745_AIO.config:148]: Does this board actually have an on-board magnetometer? If not then there's no value in setting the alignment as you have no idea how the user will install the external device.

File: configs/default/IFRC-IFLIGHT_F745_AIO.config (hunk 5 above)
Comment (Linjieqiang) [reply configs/default/IFRC-IFLIGHT_F745_AIO.config]: Thanks. Removed it now.

File: configs/default/IFRC-IFLIGHT_F745_AIO.config (hunk 6 above)
Comment (Linjieqiang) [reply configs/default/IFRC-IFLIGHT_F745_AIO.config]: This board actually have an on-board magnetometer. But I will remove align_mag flag.

Comment (etracer65) [reply configs/default/IFRC-IFLIGHT_F745_AIO.config]: I suggested removing it if the board **does not** have an onboard sensor. If the board actually has an onboard magnetometer the **you should include the alignment** that's appropriate for sensor's placement on the board.

File: configs/default/IFRC-IFLIGHT_F745_AIO.config [hunk 7]
```diff
@@ -123,11 +138,15 @@ led 3 9,11::CTOBVIW:0
 # master
 set baro_bustype = I2C
 set baro_i2c_device = 1
-set baro_hardware = BMP280
+set baro_hardware = AUTO
```
Comment (mikeller) [inline configs/default/IFRC-IFLIGHT_F745_AIO.config:141]: Please remove the default settings from the configuration.

**Review by mikeller — CHANGES_REQUESTED** [review]

File: configs/default/IFRC-IFLIGHT_F745_AIO.config (hunk 7 above)
Comment (Linjieqiang) [reply configs/default/IFRC-IFLIGHT_F745_AIO.config]: Fixed.

**Review by mikeller — APPROVED** [review]

**Pull request merged**